Tasks/DownloadPackageV1/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "downloadpackage",
+  "version": "1.0.0",
+  "description": "Download Package Task",
+  "main": "downloadPackages.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/vsts-tasks.git"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/vsts-tasks/issues"
+  },
+  "homepage": "https://github.com/Microsoft/vsts-tasks#readme",
+  "dependencies": {
+    "@types/mocha": "^2.2.5",
+    "@types/node": "^6.14.2",
+    "packaging-common": "file:../../_build/Tasks/Common/packaging-common-1.0.1.tgz",
+    "utility-common": "file:../../_build/Tasks/Common/utility-common-1.0.2.tgz",
+    "vso-node-api": "6.0.1-preview",
```

Comment:
let's take this opportunity to update this to azure-devops-node-api (6.6.0 is the version used in packaging-common)